### PR TITLE
fix issue #26

### PR DESCRIPTION
**Pull request by koudle**
fix issue #26 

The code has been code reviewed, you can see this : 
PR https://github.com/dart-lang/pub_server/pull/27

### CODE REVIEW

File: example/src/examples/http_proxy_repository.dart [hunk 1]
```diff
@@ -27,6 +27,9 @@ class HttpProxyRepository extends PackageRepository {
         baseUrl.resolve('/api/packages/${Uri.encodeComponent(package)}');
 
     http.Response response = await client.get(versionUrl);
+
+    if(response.statusCode !=200) return;
```
Comment (jonasfj) [inline example/src/examples/http_proxy_repository.dart]: Please be compliant with `dartfmt` and have whitespace between `!=` and `200`..

Just run `dartfmt`, I would recommend using visual studie code with https://dartcode.org/, it's awesome and does automatic code formatting :)

Comment (jonasfj) [reply example/src/examples/http_proxy_repository.dart]: Also please be compliant with ["Effective Dart"](https://dart.dev/guides/language/effective-dart/style#do-use-curly-braces-for-all-flow-control-structures) which says: "DO use curly braces for all flow control structures.".

In this case:
```dart
if (response.statusCode != 200) {
  return;
}
```

**Review by jonasfj — CHANGES_REQUESTED** [review]
Okay, we're almost there :)

**Comment by jonasfj** [comment]
This is really, just nits... but it's nice to be consistent :)

**Comment by jonasfj** [comment]
Also it makes travis tests fail...

File: example/src/examples/http_proxy_repository.dart (hunk 1 above)
Comment (koudle) [reply example/src/examples/http_proxy_repository.dart]: ok, thank you, I'll fix it

**Comment by koudle** [comment]
@jonasfj It's done, thanks for your advice

**Pull request merged**